tests/kafkatest/tests/streams/streams_upgrade_test.py

```diff
@@ -57,7 +57,6 @@ def perform_broker_upgrade(self, to_version):
             node.version = KafkaVersion(to_version)
             self.kafka.start_node(node)
 
-    @ignore
```

Comment:
Compare my comment on the other PR -- we might want to rewrite to replace `sleep` calls

Reply:
maybe we could merge as is and rewrite in a follow-on PR?  WDYT?

Reply:
No strong opinion -- a slight preference to keep it disabled and enable and rewrite in follow up PR. I agree, that we should exclude rewriting the test from this PR.

Reply:
I'll put the ignore annotation back until we refactor the test to remove sleep calls